docs/java/features/connectivity/destinations.md

```diff
@@ -18,6 +18,42 @@ The SAP Cloud SDK offers some basic functionality that helps with connecting to
 
 This concept is integrated with the [Destination Service](https://help.sap.com/viewer/cca91383641e40ffbe03bdc78f00f681/Cloud/en-US/7e306250e08340f89d6c103e28840f30.html) that is available on SAP Cloud Platform. If the application has a service binding to this service in place the SDK will provide access to these destinations.
 
+## Get a List of all destinations from Destination Service on Cloud Foundry  ##
```

Comment:
IMO this should be placed below the other sections. The reason is simply that it is the less common, more advanced use case that fewer people will need. It should be clear that the usual way to retrieve destinations is through the `DestinationAccessor`.

Reply:
```suggestion
## Retrieve all destinations from the Destination Service on Cloud Foundry  ##
```

Reply:
ok I agree. i will place it below.